boilerplate/app.json

```diff
@@ -7,6 +7,7 @@
     "scheme": "helloworld",
     "version": "1.0.0",
     "orientation": "portrait",
+    "userInterfaceStyle": "automatic",
```

Comment:
I think we need `expo-system-ui` installed to support this on Android

![image](https://github.com/infinitered/ignite/assets/374022/edb7f813-f57e-4013-85d3-0ef57bec9393)

Reply:
I haven't actually tested this fix explicitly in android. I'll do that and update this branch as necessary. Thanks for surfacing this!

Reply:
@frankcalise I rebased on the v10 branch and fixed a few things and I'm not seeing this when i run `npx expo prebuild --clean` in the boilerplate directory.

Reply:
Fixed in https://github.com/infinitered/ignite/pull/2636/commits/2850c86b755a91713b28bf421b57c30a06699bc1